Podfile.lock

```diff
@@ -0,0 +1,28 @@
+PODS:
```

Comment:
"Whether or not you check in the Pods directory, the Podfile and Podfile.lock should always be kept under version control."
https://guides.cocoapods.org/using/using-cocoapods.html